lib/eezee/request.rb

```diff
@@ -56,6 +58,9 @@ def setup!(options = {})
       validate!
       build_urn!
       handle_query_params!
+
+      return if preserve_url_params
+
       handle_urn_params!
```

Comment:
```suggestion
      handle_urn_params! unless @preserve_url_params
```

I think that it can be more clean